aa_forum/views/forum.py

```diff
@@ -124,6 +124,30 @@ def index(request: WSGIRequest) -> HttpResponse:
     return render(request, "aa_forum/view/forum/index.html", context)
 
 
+def _user_can_start_topic(request: WSGIRequest, current_board: Board) -> bool:
```

Comment:
How about putting this function as method on the Board model? Then it could be used in a nice OOP style, like:

```python
if board.user_can_start_topic(request.user):
    # do stuff
```